buildspec_load_test.yml

```diff
@@ -61,17 +63,21 @@ phases:
       - export AWS_ACCESS_KEY_ID=$(echo "${CREDS}" | jq -r '.Credentials.AccessKeyId')
       - export AWS_SECRET_ACCESS_KEY=$(echo "${CREDS}" | jq -r '.Credentials.SecretAccessKey')
       - export AWS_SESSION_TOKEN=$(echo "${CREDS}" | jq -r '.Credentials.SessionToken')
-      # Create and set up related testing resources
-      - python ./load_tests/load_test.py create_testing_resources
       - source ./load_tests/setup_test_environment.sh
       - export AWS_ACCESS_KEY_ID=""
       - export AWS_SECRET_ACCESS_KEY=""
       - export AWS_SESSION_TOKEN=""
       # Run load tests on corresponding platform
-      - python ./load_tests/load_test.py ${PLATFORM}
-    finally:
-      # Clear up testing resources
-      - python ./load_tests/load_test.py delete_testing_resources
+      - |
+        for i in {1..2}; do
```

Comment:
Why add retries like this? IIRC I added retries in the python code already, which I think is better than using bash.